assets/components/checkboxInput/checkboxInput.jsx

```diff
@@ -11,32 +11,44 @@ type PropTypes = {
   id: ?string,
   onChange: (preference: boolean) => void,
   checked: boolean,
-  labelText?: string,
+  labelTitle?: string,
+  labelCopy?: string,
 };
 
 
 // ----- Component ----- //
 
 export default function CheckboxInput(props: PropTypes) {
-  let label = '';
-
-  if (props.labelText && props.id) {
-    // eslint-disable-next-line jsx-a11y/label-has-for
-    label = <label htmlFor={props.id}>{props.labelText}</label>;
+  let labelTitle = '';
+  let labelCopy = '';
+
+  if (props.id) {
+    if (props.labelTitle) {
+      // eslint-disable-next-line jsx-a11y/label-has-for
```

Comment:
think you can delete these comments - they apply to the `<label>` line but also shouldn't be needed there anymore